client/luigi-client.js

```diff
@@ -279,30 +289,31 @@
       };
     },
     /**
-     * Manage UX specific options.
+     * Use the UX Manager to manage the appearance in Luigi.
      */
     uxManager: function uxManager() {
       return {
+        /** @lends uxManager */
         /**
-         * Shows a spinner with backdrop to block the micro front-end frame
+         * Adds a backdrop with a loading indicator for the micro front-end frame. This overrides the `loadingIndicator.enabled` setting.
          */
         showLoadingIndicator: function showLoadingIndicator() {
           window.parent.postMessage({ msg: 'luigi.show-loading-indicator' }, '*');
         },
         /**
-         * Hides the spinner
+         * Removes the loading indicator. Use it after calling {@link #showLoadingIndicator showLoadingIndicator()} or to hide the indicator when you use the `loadingIndicator.hideAutomatically: false` Node configuration.
```

Comment:
Can loadingIndicator.hideAutomatically be linked to the configuration?

Reply:
Linked to `navigation-configuration.md#nodes`